playbooks/roles/foreman_installer/tasks/module_pr.yml

```diff
@@ -12,6 +12,22 @@
     - yard
     - puppet-strings
   when: puppetlabs_gem.stat.exists
+  tags:
+    - packages
+
+- name: 'install yard gem'
```

Comment:
This is no longer needed, the above code coverages this. It has been refactored.